cloudmarker/test/test_firewallevent.py

```diff
@@ -0,0 +1,82 @@
+"""Tests for Firewall Event Plugin."""
+
+import copy
+import unittest
+
+from cloudmarker.events import firewallevent
+
+mock_firewall_record = {
+    'kind': 'compute#firewall',
+    'id': '7890789078907890',
+    'creationTimestamp': '2018-12-19T01:43:51.988-08:00',
+    'name': 'default-allow-ssh',
+    'description': 'str',
+    'network': 'https://www.googleapis.com/compute/v1/\
+projects/foo/global/networks/default',
```

Comment:
Please rewrite this as:

```
    'network': 'https://www.googleapis.com/compute/v1/'
               'projects/foo/global/networks/default',
```

Two or more consecutive string literals next to each other coalesce to form a single string literal. This feature exists in most programming languages. This feature is very useful to split a long string into multiple lines while using proper indentation of our choice.

Please make this change in lines 37, 47, and 49 below too.